src/components/MovieList/MovieList.jsx

```diff
@@ -1,5 +1,8 @@
 import './MovieList.scss';
+import { MovieCard } from '../MovieCard';
 
-export const MovieList = () => (
-  <>Put the list here</>
+export const MovieList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => <MovieCard movie={movie} key={movie.imdbId} />)}
```

Comment:
I suggest moving <MovieCard /> and its' props to new line like this:
movies.map(movie => (
MovieCard movie={movie} key={movie.imdbId}
    ))
 But don't forget opening and closing tags